plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployCommandHandler.java

```diff
@@ -69,6 +80,18 @@ public void done(IJobChangeEvent event) {
     deploy.schedule();
   }
 
+  private IPath createWorkDirectory() throws IOException {
+    String now = Long.toString(System.currentTimeMillis());
+    IPath workDirectory = getTempDir().append(now);
+    Files.createDirectories(workDirectory.toFile().toPath(), new FileAttribute<?>[0]);
```

Comment:
Huh. I must be missing something about how varargs work then. I thought you could simply pass no arguments for varargs. The Javadoc says it's optional:

https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createDirectories(java.nio.file.Path,%20java.nio.file.attribute.FileAttribute...)